packages/runtime-node/package.json

```diff
@@ -29,4 +29,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
+}
```

Comment:
how do you change these files? do you run `pleb upgrade?`

Reply:
not in this chase, since I wanted to only update core3utils libs